.github/workflows/cli.yml

```diff
@@ -0,0 +1,74 @@
+name: Run Tests
+
+on:
+  pull_request:
+    branches:
+      - master
+      - v1
+
+env:
+  GO111MODULE: on
+  GOPROXY: https://proxy.golang.org
+
+jobs:
+  test:
+    strategy:
+      matrix:
+        os: [ubuntu-latest, macos-latest]
+        go: [1.11, 1.12, 1.13]
+    name: ${{ matrix.os }} @ Go ${{ matrix.go }}
+    runs-on: ${{ matrix.os }}
+    steps:
+      - name: Set up Go ${{ matrix.go }}
+        uses: actions/setup-go@v1
+        with:
+          go-version: ${{ matrix.go }}
+
+      - name: Set GOPATH and PATH
+        run: |
+          echo "##[set-env name=GOPATH;]$(dirname $GITHUB_WORKSPACE)"
+          echo "##[add-path]$(dirname $GITHUB_WORKSPACE)/bin"
+        shell: bash
+
+      - name: Checkout Code
+        uses: actions/checkout@v1
+        with:
+          ref: ${{ github.ref }}
+
+      - name: Install Dependencies (Linux)
+        if: matrix.os == 'ubuntu-latest'
+        run: |
+          mkdir -p $GOPATH/bin
+          curl -s https://api.github.com/repos/urfave/gfmrun/releases/latest | grep gfmrun-linux-amd64 | grep download | cut -d '"' -f 4 | wget -O $GOPATH/bin/gfmrun -i -
```

Comment:
In case `gfmrun` ever makes a breaking change, we'd be better off pinning the dependency version. This way, we can control exactly when our pipeline breaks.

We can also avoid keeping two different install sections if we want by relying on `go env` to tell us what OS we're dealing with. These two things together give:

```bash
wget -O $GOPATH/bin/gfmrun https://github.com/urfave/gfmrun/releases/download/v1.2.14/gfmrun-$(go env GOOS)-amd64-v1.2.14
```

Reply:
I doubt ‘gfmrun’ would make such a change as I’m one of the maintainers of ‘gfmrun’ but this makes sense